pkg/kn/commands/broker/list.go

```diff
@@ -43,7 +43,8 @@ func NewBrokerListCommand(p *commands.KnParams) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List brokers",
+		Short:   "List brokers(Alias: 'ls')",
```

Comment:
Doesn't Cobra usually include Aliases under `-h` without the need to include in the command description? There does appear to be something preventing it with `kn`, but it would probably be easier to maintain than documenting like this. 

A nit is that there should probably be a space between `List brokers` and the aliases if keeping this approach is preferred:

```suggestion
		Short:   "List brokers (Alias: 'ls')",
```

Reply:
help using cobra shows plurals? If yes then it might be confusing.

Reply:
Will revisit this as part of #1045. Resolving this here.